src/ResourceManager/NotificationHubs/Commands.NotificationHubs/NotificationHubsManagementClient.cs

```diff
@@ -241,6 +240,10 @@ public NotificationHubAttributes UpdateNotificationHub(string resourceGroupName,
                 RegistrationTtl = nhAttributes.RegistrationTtl
             };
 
+            if (nhAttributes.Tags != null)
```

Comment:
Can we add a test to verify that this fixes the issue, and prevent future regressions